vaapi/vaapidisplay_unittest.cpp

```diff
@@ -103,14 +103,9 @@ class VaapiDisplayTest : public ::testing::Test {
 
 #if __ENABLE_X11__
 VAAPI_DISPLAY_TEST(CreateInvalidHandleX11) {
-    FAIL(); // remove when fixed;
-
-    // FIXME: This causes segmentation fault.
-    // VaapiDisplay should be able to handle this gracefully
     NativeDisplay native = {1, NATIVE_DISPLAY_X11};
```

Comment:
Gtest uses fork for death tests by default which it claims is unsafe if we are using threads in the library.  For some reason, gtest is "unable to detect" whether we are using threads in the library so it warns us.

AFAICT, instructing gtest to just use a thread for death tests instead of fork is harmless and better.  That is what the above code statement does.